packages/storybook-addon-baseline-grid/src/components/Tool.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { memo, useCallback, useEffect } from "react";
+import { useGlobals, type API } from "storybook/internal/manager-api";
+import { IconButton } from "storybook/internal/components";
+import { KEY, TOOL_ID } from "../constants";
+import { MenuIcon } from "@storybook/icons";
+
+export const Tool = memo(function MyAddonSelector({ api }: { api: API }) {
+  const [globals, updateGlobals, storyGlobals] = useGlobals();
+
+  const isLocked = KEY in storyGlobals;
+  const isActive = !!globals[KEY];
+
+  const toggle = useCallback(() => {
+    updateGlobals({
+      [KEY]: !isActive,
+    });
+  }, [isActive, updateGlobals]);
+
+  // TODO: can be used to add keyboard shortcut
+  // useEffect(() => {
+  //   api.setAddonShortcut(ADDON_ID, {
+  //     label: "Toggle Baseline Grid",
+  //     defaultShortcut: ["O"],
+  //     actionName: "baseline",
+  //     showInMenu: false,
+  //     action: toggle,
+  //   });
+  // }, [toggle, api]);
+
+  return (
+    <IconButton
+      key={TOOL_ID}
+      active={isActive}
+      disabled={isLocked}
+      title="Show baseline grid"
+      onClick={toggle}
+    >
+      <MenuIcon />
+    </IconButton>
```

Comment:
We now have two baseline buttons in the react core storybook: one that doesn't, and one that does.

![Screenshot 2024-12-19 at 16 26 06](https://github.com/user-attachments/assets/5cee4fa1-a439-4997-b235-657d0db832d7)

Can we remove the old button, and consider giving the new button an always-visible label as well (like the old button says "Baseline visible/Baseline hidden"?

Reply:
Done.